requirements/200.md

```diff
@@ -0,0 +1,54 @@
+# Level 200 - Week 2 - UX Improvements
+
+**Overview**
+
+Following feedback on the MVP, this iteration focuses on enhancing user experience through embedded videos, improved design, and accessibility considerations.
```

Comment:
Would be good to know whether we want to have a "dry" approach to the descriptions and user stories or a more example based approach. The old level 200 description (and the original PR) aimed more for the example based approach like the following:

```
Your MVP has been well received by your friends, but they found it lacking in some regards. The main comments you received were:

- They found clicking on the link and getting redirected was not convenient, they got lost while opening up the tabs
- They said the styling of the page was not too welcoming, and suggested you add some colours to it.
```

This has been retained in later levels, like the one in level 210. I generally feel we should keep a more lighthearted style, but if that's not how we should do we should change the other levels to be more dry as well.

Reply:
Keep the jokes ;)

Reply:
@sztupy The requirements you documented in levels have now been captured as issues on this board, in line with the structure of the other module products: https://github.com/orgs/CodeYourFuture/projects/169/views/2
I've put some of the lighter material back into the issues but if you want to check/add more, add them back to the issues, thankyou!